src/remarx/quotation/find_quotes.py

```diff
@@ -12,9 +22,35 @@
 import sys
 
 from remarx.quotation.pairs import find_quote_pairs
-from remarx.utils import configure_logging
+from remarx.utils import configure_logging, get_default_corpus_path
 
 logger = logging.getLogger(__name__)
+_, DEFAULT_CORPUS_DIRS = get_default_corpus_path()
```

Comment:
It's better to init in `main` at run time than here at load time. I'm going to update to use `CorpusPath` class directly.